prisma/schema.prisma

```diff
@@ -100,11 +100,11 @@ model trees_watered {
   time      String
   uuid      String?
   amount    String?
-  timestamp DateTime? @db.Timestamp(6)
+  timestamp DateTime @db.Timestamptz()
```

Comment:
Did you get this from doing a `npx prisma db pull` or did you do it manually?

Reply:
Originally manually. Now I did a prisma db pull and it changed it to `Timestamptz(6)` which should be right.